app/views/stages/_fields.html.erb

```diff
@@ -23,8 +23,10 @@
       " When selected with '#{no_code_label}' and '#{confirm_label}' is disabled the stage can be directly executed."
   %>
 
-  <% help = "Bypass " + [("buddy check" if BuddyCheck.enabled?), "release tracking"].compact.to_sentence %>
-  <%= form.input :no_code_deployed, as: :check_box, label: no_code_label, help: help %>
+  <% help = "Bypass " + [("buddy check" if BuddyCheck.enabled?), "release tracking"].compact.to_sentence \
+      + ". Must be a global admin to change"
```

Comment:
Missing period

Reply:
it's missing for all other help text? I thought to make it consistency.

Reply:
hmm.. will add for remaining too.